test/perceptual/.eslintrc.json

```
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-invalid-this": 0
```

```suggestion
    "no-invalid-this": "off"
```

Recommend using `off` for better readability